dkist/dataset/tiled_dataset.py

```diff
@@ -153,10 +153,12 @@ def tiles_shape(self):
         return [[tile.data.shape for tile in row] for row in self]
 
     def plot(self, slice_index: int, share_zscale=False, **kwargs):
```

Comment:
What is `slice_index` now? is it actually a generic slice which should lead to a spatial-spatial plot?

Also this method needs a docstring.

Reply:
It's an int or tuple that specifies the slice of each tile to plot, ie it's equivalent to the input of `slice_tiles`. That will give a spatial-spatial plot if those are the last two pixel dimensions, but obviously that's not guaranteed. I guess in principle if I generalise it to take an actual slice object as well then it will be able to plot whatever arrangement of coords you want, but I'm not sure how much physical sense that makes for multi-dimensional tiles.

Reply:
I think this would work if you did give it a full output of `np.s_`? I think this probably needs documenting well and then testing a bunch so we understand any quirks here.